tcms/telemetry/templates/telemetry/testing/breakdown.html

```diff
@@ -7,56 +7,7 @@
 {% block contents %}
 
     <div class="container-fluid container-cards-pf">
-        <form class="form-horizontal">
-            <div class="form-group">
-                <div class="fa pficon-help help-tooltip" data-toggle="tooltip" data-placement="left" title="{% trans 'Shows how existing test cases are distributed' %}"></div>
-            </div>
-            <div class="form-group">
-                <label for="id_product" class="col-md-1 col-lg-1">
-                    {% trans "Product" %}:
-                </label>
-                <div class="col-md-3">
-                    <select name="product" id="id_product" class="form-control selectpicker">
-                        <option value="">----------</option>
-                    </select>
-                </div>
-
-                <label for="id_test_plan" class="col-md-1 col-lg-1">
-                    {% trans "Test Plan" %}:
-                </label>
-                <div class="col-md-3">
-                    <select name="test_plan" id="id_test_plan" class="form-control selectpicker">
-                        <option value="">----------</option>
-                    </select>
-                </div>
-            </div>
-
-            <div class="form-group">
-                <label class="col-md-1 col-lg-1" for="id_after">{% trans "After" %}</label>
-                <div class="col-md-3 col-lg-3">
-                    <div class="input-group date-time-picker-pf">
-                        <input type="text" class="form-control" id="id_after">
-                        <span class="input-group-addon">
-                            <span class="fa fa-calendar"></span>
-                        </span>
-                    </div>
-
-                    {% include "datetimepicker_script.html" with selector="#id_after" %}
-                </div>
-
-                <label class="col-md-1 col-lg-1" for="id_before">{% trans "Before" %}</label>
-                <div class="col-md-3 col-lg-3">
-                    <div class="input-group date-time-picker-pf">
-                        <input type="text" class="form-control" id="id_before">
-                        <span class="input-group-addon">
-                            <span class="fa fa-calendar"></span>
-                        </span>
-                    </div>
-
-                    {% include "datetimepicker_script.html" with selector="#id_before" %}
-                </div>
-            </div>
-        </form>
+        {% include "telemetry/filters.html" %}
```

Comment:
Use `telemetry/include` for the dir name b/c we already have this naming in the high-level `templates/include` directory. Also makes it very clear what's inside.